### ISSUE TITLE
rename selections to all-selections ?

### ISSUE THREAD

**Issue by jonenst** [factor/factor#2388]
These are related but don't follow the same naming convention:
```
{ -1 0 1 } 2 all-combinations .
{ -1 0 1 } 2 selections .
```

```
{
    { -1 0 }
    { -1 1 }
    { 0 1 }
}
{
    { -1 -1 }
    { -1 0 }
    { -1 1 }
    { 0 -1 }
    { 0 0 }
    { 0 1 }
    { 1 -1 }
    { 1 0 }
    { 1 1 }
}
```

**Comment by mrjbq7** [comment]
Would you prefer ``all-selections``?

**Comment by jonenst** [comment]
Yes that's what I was proposing :)

**Comment by mrjbq7** [comment]
Fixed in 1fdcb625b705d353b883d8256af366269507a97d.
